src/payment/strategies/ppsdk/step-handler/error.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { RequestError } from '../../../../common/error/errors';
```

Comment:
Currently `error`, `failed` and `networkError` all do the exact same thing.

For now I wanted to keep the as seperate (though near identical) branches of code until further work is done to decide how the PPSDK strategy should act in each of these scenarios.

At which point they'll have either diverged or combined as required.